client/js/components/week/week.js

```diff
@@ -10,26 +10,28 @@ var moment = require('moment');
 var _ = require('underscore');
 
 var Week = React.createClass({
-  // weekData is a temporary mock in place prior to server and db setup
   getInitialState: function () {
     return {};
   },
+
   componentDidMount: function () {
     WeekStore.addChangeListener(this.handleStoreChange);
-    ViewActionCreators.getWeekData();
+    ViewActionCreators.getWeekData(moment());
   },
+
   handleStoreChange: function () {
-    this.setState(WeekStore.getWeekData());
-  },
-  render: function () {
-    var days = _.groupBy(this.state, function (day) {
-      return moment(day.date).format('YYYYMMDD');
+    var weekData = WeekStore.getWeekData();
```

Comment:
thoughts on formatting this weekData in the store itself, when it is received as a payload from the dispatcher